.test-infra/jenkins/LoadTestsBuilder.groovy

```diff
@@ -25,6 +25,7 @@ import static PythonTestProperties.LOAD_TEST_PYTHON_VERSION
 
 class LoadTestsBuilder {
   final static String DOCKER_CONTAINER_REGISTRY = 'gcr.io/apache-beam-testing/beam_portability'
+  final static String DOCKER_CONTAINER_REGISTRY_GO = 'gcr.io/apache-beam-testing/beam-sdk'
```

Comment:
So, I've moved it to a variable, but I'll also point out that there are so many ways these configurations could be refactored to avoid all the repetition, while allowing for one off changes. So further changes of this nature are out of scope for this PR. 

Eg. each specific load test has it's configuration repeated in the three languages, with very few changes between them other than the SDK container, and the name.

Reply:
I agree a refactor would be nice eventually, I like where you landed for this PR though